hypha/apply/templates/forms/includes/field.html

```diff
@@ -56,7 +56,11 @@
 
                 {{ field }}
 
-                {% if field.errors %}<h6 class="form__error-text">{{ field.errors.as_text|linebreaksbr }}</h6>{% endif %}
+                {% if field.errors %}
+                    <h6 class="form__error-text">
```

Comment:
Can we get rid of the "h6" as well. The original Hypha theme misuse headings quite a lot.